doc/ref_model/chapters/chapter07.md

```diff
@@ -311,40 +348,45 @@ The static file system includes a set of important files and folders which do no
 
-* Confidentiality – Only allow access to data for which the user is permitted.
-* Integrity – Ensure data is not tampered with or altered by unauthorized users.
-* Availability – ensure systems and data are available to authorized users when they need it.
+The strength of open source code is the availability of code source developed by a community which maintain and improve it. Open source code integration with application source code helps to develop and produce applications faster. But, in return, it can introduce security risks if a risk management DevSecOps approach is not implemented. The GSMA white paper, “Open Networking & the Security of Open Source Software Deployment - Future Networks”, alerts on these risks and addresses the challenges coming with open source code usage. Amongst these risks for security, we can mention a poor quality code containing security flaws, an obsolete code with known vulnerabilities, and the lack of knowledge of open source communities’ branches activity. An active branch will come with bugs fixes, it will not be the case with an inactive branch. The GSMA white paper develops means to mitigate these security issues.
 
-Additional Cloud Infrastructure security principles that need to be incorporated:
-* Authenticity – The ability to confirm the users are in fact valid users with the correct rights to access the systems or data.
+**SBOM**
 
-In mobile network field, the GSM Association ([GSMA](https://www.gsma.com/)) and its Fraud and Security working group of experts have developed a set of documents specifying how to secure the global mobile ecosystem. 
-* The document “Baseline Security controls”, [FS.31 v2.0](https://www.gsma.com/security/resources/fs-31-gsma-baseline-security-controls/), published in February 2020, is a practical guide intended for operators and stakeholders to check mobile network’s internal security. It lists a set of security controls from business controls (including security roles, organizational policies, business continuity management…) to technological controls (for user equipment, networks, operations…) covering all areas of mobile network, including Cloud Infrastructure. A checklist of questions allows to improve the security of a deployed network. 
+To begin, it is highly recommended to identify the software components and their origins. The Software Bill of Materials (SBOM), described by [US NTIA](https://www.ntia.gov/SBOM)(National Telecommunications and Information Administration), is an efficient tools to identify software components. The SBOM is an inventory of software components and the relationships between them. NTIA describes how to establish an SBOM and provides SBOM standard data formats. In case of vulnerability detected for a component, the SBOM inventory is an effective means to identify the impacted component and provide remediation.
```

Comment:
```suggestion
To begin, it is highly recommended to identify the software components and their origins. The Software Bill of Materials (SBOM), described by [US NTIA](https://www.ntia.gov/SBOM)(National Telecommunications and Information Administration), is an efficient tool to identify software components. The SBOM is an inventory of software components and the relationships between them. NTIA describes how to establish an SBOM and provides SBOM standard data formats. In case of vulnerability detected for a component, the SBOM inventory is an effective means to identify the impacted component and provide remediation.
```